components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/device/dao/DeviceFlowDAO.java

```diff
@@ -30,6 +31,23 @@
  */
 public interface DeviceFlowDAO {
 
+    /**
+     * This will be used to enter the values to the database tables and return the unique user_code.
+     *
+     * @param deviceCode  Code that is used to identify the device.
+     * @param userCode    Code that is used to correlate user and device.
+     * @param quantifier  Quantized time period user_code belongs.
+     * @param consumerKey Consumer key of the client application.
+     * @param scopes      Requested scopes.
+     * @return Unique user_code.
+     * @throws IdentityOAuth2Exception Error while inserting device flow parameters.
+     */
+    default String insertDeviceFlowParametersWithQuantifier(String deviceCode, String userCode, long quantifier,
+        String consumerKey, String scopes) throws IdentityOAuth2Exception {
+
+        throw new NotImplementedException("Not Implemented");
```

Comment:
```suggestion
        throw new NotImplementedException("Not Implemented.");
```
Change in other places as well.

Reply:
fixed